gtsfm/bundle/bundle_adjustment.py

```diff
@@ -136,6 +145,37 @@ def create_computation_graph(self, sfm_data_graph: Delayed) -> Delayed:
         return dask.delayed(self.run)(sfm_data_graph)
 
 
+def aggregate_ba_stats(ba_data: GtsfmData) -> Dict[str,Any]:
+    """Create a dictionary of summary statistics for a bundle adjustment result.
+
+    Args:
+        ba_data: bundle adjustment result
+
+    Returns:
+        dictionary containing summary statistics of bundle adjustment result
```

Comment:
We probably also need to align on some terminology here. Summary, statistics and metrics. Could we define each one so we be precise across gtsfm? 
I have been using metric for a quantity, and statistic for min, max, etc. Technically the statistics are also metrics so its not super accurate.